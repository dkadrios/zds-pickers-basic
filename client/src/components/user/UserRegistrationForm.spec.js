// import set from 'lodash/fp/set'
// import { store, user } from '../../../__mocks__'
// import actionTypes from '../../reducers/actionTypes'
// import { UserRegistrationForm, mapDispatchToProps, mapStateToProps } from './UserRegistrationForm'

// const handleSubmit = jest.fn()

// const props = {
//   user,
//   handleSubmit,
//   error: '',
//   classes: {},
// }

// describe('UserRegistrationForm tests', () => {
//   beforeEach(jest.clearAllMocks)

//   it('renders correctly', () => {
//     shallowExpect(UserRegistrationForm)(props).toMatchSnapshot()
//   })

//   it('renders errors correctly', () => {
//     shallowExpect(UserRegistrationForm)(set('error', 'error message')(props)).toMatchSnapshot()
//   })

//   it('should utilize mapStateToProps', () => {
//     expect(mapStateToProps(store)).toEqual({ initialValues: user })
//   })

//   it('should utilize mapDispatchToProps', () => {
//     const dispatch = jest.fn()
//     mapDispatchToProps(dispatch).showDialog()
//     expect(dispatch.mock.calls[0][0]).toEqual({ type: actionTypes.SHOW_REGISTRATION_DLG })
//   })
// })
