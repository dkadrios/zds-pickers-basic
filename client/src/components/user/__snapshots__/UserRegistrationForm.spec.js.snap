// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserRegistrationForm tests renders correctly 1`] = `
<form
  onSubmit={[MockFunction]}
>
  <Field
    component={[Function]}
    label="First Name"
    name="firstName"
    type="text"
    validate={
      Array [
        [Function],
        [Function],
      ]
    }
  />
  <Field
    component={[Function]}
    label="Last Name"
    name="lastName"
    type="text"
    validate={
      Array [
        [Function],
        [Function],
      ]
    }
  />
  <Field
    component={[Function]}
    label="Email"
    name="email"
    type="email"
    validate={
      Array [
        [Function],
        [Function],
      ]
    }
  />
</form>
`;

exports[`UserRegistrationForm tests renders errors correctly 1`] = `
<form
  onSubmit={[MockFunction]}
>
  <Field
    component={[Function]}
    label="First Name"
    name="firstName"
    type="text"
    validate={
      Array [
        [Function],
        [Function],
      ]
    }
  />
  <Field
    component={[Function]}
    label="Last Name"
    name="lastName"
    type="text"
    validate={
      Array [
        [Function],
        [Function],
      ]
    }
  />
  <Field
    component={[Function]}
    label="Email"
    name="email"
    type="email"
    validate={
      Array [
        [Function],
        [Function],
      ]
    }
  />
  <strong>
    error message
  </strong>
</form>
`;
